core/ir/riscv64/codec.c

```diff
@@ -486,7 +486,7 @@ decode_u_immpc_opnd(dcontext_t *dc, uint32_t inst, int op_sz, byte *pc, byte *or
                     int idx, instr_t *out)
 {
     int32_t uimm = GET_FIELD(inst, 31, 12);
-    opnd_t opnd = opnd_create_pc(orig_pc + (uimm << 12));
+    opnd_t opnd = opnd_create_rel_addr(orig_pc + (uimm << 12), OPSZ_0);
```

Comment:
OPSZ_0 because we we're not really getting any data from the addr, but only a pc? Maybe add a comment.

Reply:
Added a comment